### PR TITLE
armadillo changed their library directory in recent versions from lib…

### PR DESCRIPTION
**Pull request by mboisson**
… to lib64

This patch fixes that.

### CODE REVIEW

**Comment by mboisson** [comment]
@boegel Mmmm, the failed test seems unrelated to this PR

**Comment by boegel** [comment]
failing tests fixed by https://github.com/easybuilders/easybuild-framework/pull/2258, I'll retrigger them

File: easybuild/easyblocks/a/armadillo.py [hunk 1]
```diff
@@ -56,8 +56,13 @@ def configure_step(self):
...
     def sanity_check_step(self):
         """Custom sanity check for Armadillo."""
+        if LooseVersion(self.version) < LooseVersion('7.950.1'):
+            libdir = 'lib'
+        else:
+            libdir = 'lib64'
+
         custom_paths = {
-            'files': ['include/armadillo', 'lib/libarmadillo.%s' % get_shared_lib_ext()],
+            'files': ['include/armadillo', libdir + '/libarmadillo.%s' % get_shared_lib_ext()],
```
Comment (boegel) [inline easybuild/easyblocks/a/armadillo.py]: more Python-esque:

```python
os.path.join(libdir, 'libarmadillo.%s' % get_shared_lib_ext())
```

**Comment by boegel** [comment]
@mboisson Do you have a PR for an easyconfig for a recent version of Armadillo that requires this?

**Comment by boegel** [comment]
nvm, found it: https://github.com/easybuilders/easybuild-easyconfigs/pull/4849

**Comment by mboisson** [comment]
Done

**Comment by boegel** [comment]
Tested with existing Armadillo easyconfigs & https://github.com/easybuilders/easybuild-easyconfigs/pull/4849, good to go, thanks @mboisson!

**Pull request merged**